vm/vmExpr.go

```diff
@@ -737,12 +737,12 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
 	case *ast.DeleteExpr:
 		mapExpr, err := invokeExpr(e.MapExpr, env)
 		if err != nil {
-			return nilValue, newStringError(e, err.Error())
+			return nilValue, newStringError(e.MapExpr, err.Error())
```

Comment:
Sorry, I updated my comment to slow, this should be:
return nilValue, newError(e.MapExpr, err)

Reply:
done.